e2e/benchmarks/local-benchmark/index.html

```diff
@@ -655,10 +661,19 @@ <h2>TensorFlow.js Model Benchmark</h2>
       return false;
     }
 
+    function isBackendSupported() {
+      if (state.backend === 'webgpu' && typeof navigator !== 'undefined' && !navigator.gpu){
+        showMsg(`These samples should run in Chrome Canary behind the flag "--enable-unsafe-webgpu".`);
+        throw new Error(`WebGPU is not supported.`);
+      }
+      showMsg(null);
+    }
```

Comment:
Put it in `index.js` and define it like below:
```
function isBackendSupported(backendName) {
  if (backendName === 'webgpu' && typeof navigator !== 'undefined' &&
      !navigator.gpu) {
    return false;
  } else {
    return true;
  }
}

Reply:
At the beginning of `testCorrectness` and `runBenchmark`, insert below code so that we can early return:
```
      if(!isBackendSupported(state.backend))
      {
              return;
      }
```

And when switch backend, do it like below:
```
backendsController.onChange(async backend => {
        if (!isBackendSupported(state.backend))
        {
          showMsg(
        `WebGPU is not supported. Please use Chrome Canary browser with flag "--enable-unsafe-webgpu" enabled.`);
          return;
        } else {
           showMsg(null);
       }
```
And in other places, you don't need to call isBackendSupported.